backend/src/main/java/gov/cdc/usds/simplereport/service/SupportedDiseaseService.java

```diff
@@ -0,0 +1,21 @@
+package gov.cdc.usds.simplereport.service;
+
+import gov.cdc.usds.simplereport.db.model.SupportedDisease;
+import gov.cdc.usds.simplereport.db.repository.SupportedDiseaseRepository;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class SupportedDiseaseService {
```

Comment:
for this class, I was wondering if we would want to combine it with DiseaseService, which is being added in #3560

Reply:
Aha -  good catch, I missed that we are both adding this service. We'll have to keep that in mind, I will changed the name of my service to `DiseaseService`, then whichever PR gets merged first, the other will have some merge conflicts, but they should be pretty easy to merge.